.copywrite.hcl

```diff
@@ -12,6 +12,7 @@ project {
     "test/fixtures/**",
```

Comment:
Seems like we should update this line and the one above? (And remove the headers that were added to the test files)